README.md

```diff
@@ -35,46 +53,22 @@ Prerequisites to Run Integration Tests
             * `repo`
             * `admin:repo_hook`
             * `delete_repo`
-    * Create 3 environment variables:
-        * `GITHUB_USERNAME`
-        * `GITHUB_TOKEN`
-        * `GITHUB_PASSWORD`
+    * Create 2 environment variables:
+        * `CATAPULT_GITHUB_USERNAME`
+        * `CATAPULT_GITHUB_TOKEN`
 
     For instance you may put into your `~/.bash_profile`:
 
-        export GITHUB_USERNAME=ALRubinger
-        export GITHUB_TOKEN=[token created from above]
-        export GITHUB_PASSWORD=mygithub-password
+        export CATAPULT_GITHUB_USERNAME=<your github username>
+        export CATAPULT_GITHUB_TOKEN=<token created from above>
     
-    Updates to `~./bash_profile` may require logging out of the shell to be visible; you may check by typing into a terminal:
+    Use `source ~./bash_profile` to make your changes visible; you may check by typing into a terminal:
```

Comment:
Should now be covered by `catapult-env.sh` above, no longer in `.bash_profile`, right?